utils/parse.ts

```diff
@@ -0,0 +1,11 @@
+export const extractTwitterIdFromDescription = (desc: string = '') => {
+  const regex =
+    /(?:https?:\/\/)?(?:www\.)?twitter\.com\/(?:\#!\/)?@?([^\/\?\s)]+)/
```

Comment:
please update regex to make `https://x.com/` valid

Reply:
done